src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1202,6 +1203,13 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists('Symfony\Component\Serializer\Normalizer\ConstraintViolationListNormalizer')) {
```

Comment:
No, I want to be consistent with other usage in this file. just above and bellow